maestro-driver-ios/maestro-driver-iosUITests/Routes/ServerError.swift

```diff
@@ -0,0 +1,6 @@
+enum ServerError: Error {
```

Comment:
Lets remove this completely I have also stopped throwing errors since all of them should be client errors. See the hierarchy handler for implementation